tox/session.py

```diff
@@ -627,8 +627,9 @@ def showconfig(self):
                                  % (attr.name, getattr(envconfig, attr.name)))
 
     def showenvs(self):
+        max_length = max(len(env) for env in self.config.envlist)
         for env in self.config.envlist:
-            self.report.line("%s" % env)
+            self.report.line("{} {}".format(env.ljust(max_length), self.config.envconfigs[env].description))
```

Comment:
This should be `{0} {1}` to work on Python 2.6 - it seems like it doesn't get caught in the tests though, and they report "can not run tests involving calling tox on python2.6. (and python2.6 is about to be deprecated anyway)"...